app/src/components/pages/divisionsPage/divisionsPage.tsx

```diff
@@ -12,7 +12,7 @@ import { DC_URL } from "@/src/config/constants";
 import CollectionLanesLoading from "../../collectionLanes/collectionLanesLoading";
 
 export default function DivisionsPage({ data }) {
-  console.log("clientside divisions", data);
+  console.log("clientside divisions", data.divisions.length);
```

Comment:
I can't run this locally right now but I'm getting this run time error:
<img width="688" alt="Screenshot 2024-09-23 at 10 34 10 AM" src="https://github.com/user-attachments/assets/124c2215-e905-42af-bd8e-42ea0636e8aa">

This will be good to catch and return a default `{ divisions: [] }` from the server component.